src/components/profile/profile.tsx

```diff
@@ -67,13 +202,56 @@ const Profile = () => {
             .
           </p>
         </div>
-        <div style={{ margin: "1em 0" }}>
-          <Alert variant={user?.timeout ? "success" : "danger"}>
-            {user.timeout
-              ? "Your user session is active."
-              : "Your session has expired!"}
-          </Alert>
-        </div>
+        {userIVAs.length > 0
+          ? userIVAs.map((x, index) => {
+              return (
+                <Row key={x.id + index}>
+                  <Col xs={3}>
+                    {IVAType[x.type]}: {x.value}
+                  </Col>
+                  <Col xs={2}>
+                    {x.status === IVAStatus.Unverified ? (
+                      <Button
+                        id={"del_" + x.id}
+                        className="p-0 px-1 bg-quinary"
+                        onClick={(e) => {
+                          let button = e.target as HTMLButtonElement;
+                          HandleRequestVerification(button.id.substring(4));
+                        }}
+                      >
+                        Request verification
+                      </Button>
+                    ) : x.status === IVAStatus.CodeTransmitted ? (
+                      <Button
+                        id={"con_" + x.id}
+                        className="p-0 px-1 bg-quinary"
+                        onClick={(e) => {
+                          let button = e.target as HTMLButtonElement;
+                          HandleConfirmVerification(button.id.substring(4));
+                        }}
+                      >
+                        Enter verification code
+                      </Button>
+                    ) : (
+                      IVAStatus[x.status]
```

Comment:
This needs translation to human readable form as well, e.g. "verification requested" instead of "VerificationRequested".

Reply:
Would be good to color code the different statuses. Green for ok (verified), and orange for states that indicate something still needs to be done (i.e. everything else).

Reply:
Done 👍